Misc/python-config.in

```diff
@@ -35,10 +35,10 @@ if '--help' in opt_flags:
 
```

Comment:
You can replace "pyver = sysconfig.get_config_var('VERSION')" with "pyver = getvar('VERSION')" :-)